packages/field/src/FormControlMixin.js

```diff
@@ -574,5 +588,9 @@ export const FormControlMixin = dedupeMixin(
       addToAriaDescription(id) {
         this._ariaDescribedby += ` ${id}`;
       }
+
+      __getDirectSlotChild(slotName) {
```

Comment:
This helper will be needed in every component querying slots. Maybe add to SlotMixin?